### DISCUSSION TITLE
stuck on awaiting data

### DISCUSSION THREAD

**Discussion by gioAdu** [vercel/community#3420] [Help]
### Summary

i'm on hobby/free plan , using Nextj13 as framework. i've uploaded the project enabled speed insights and redeployed it. visited the page did some activities , still seems to be stuck on awaiting data . not using any CSP

### Steps to Reproduce

Not really much to replicate, other than me going in and enabling the feature.

**Comment by janoschherrmann** [comment]
I have a very similar situation, but on a Svelte / SvelteKit project. It says the project is ready, but nothing happens after that, even when I try to access the page multiple times, still no insights.
